backend/index.js

```diff
@@ -28,13 +31,19 @@ app.set('view engine', 'jade');
 //Route utilization
 app.use('/', indexRouter);
 app.use('/users', usersRouter);
+app.use('/patients', patientsRouter);
+
+// CORS headers
+app.use(function (req, res, next) {
```

Comment:
is this needed? I thought only having `app.use(cors());` from line 14 was required